src/components/Overview/IntroBanner.tsx

```diff
@@ -18,7 +18,6 @@ import { AccessReviewResources } from '../../types';
 import { useAccessReviewForModels } from '../../utils/rbac';
 import { ButtonWithAccessTooltip } from '../ButtonWithAccessTooltip';
 import { SignupStatus } from '../SignUp/signup-utils';
-import SignupButton from '../SignUp/SignupButton';
 import { useSignupStatus } from '../SignUp/useSignupStatus';
```

Comment:
Remove other SignUp related components and utilities.

Reply:
But we still use this right? Only not through button.

Reply:
No , we dont use it.